src/pages/settings/user/components/CustomFields.tsx

```diff
@@ -22,17 +23,21 @@ export function CustomFields() {
   const handleCustomFieldChange = useHandleCustomFieldChange();
 
   return (
-    <Card title={t('custom_fields')}>
-      {company &&
-        ['user1', 'user2', 'user3', 'user4'].map((field) => (
-          <Field
-            key={field}
-            initialValue={company.custom_fields[field]}
-            field={field}
-            placeholder={t('user_field')}
-            onChange={(value: any) => handleCustomFieldChange(field, value)}
-          />
-        ))}
-    </Card>
+    <>
+      <CustomFieldsPlanAlert />
+
+      <Card title={t('custom_fields')}>
+        {company &&
+          ['user1', 'user2', 'user3', 'user4'].map((field) => (
+            <Field
+              key={field}
+              initialValue={company.custom_fields[field]}
+              field={field}
+              placeholder={t('user_field')}
+              onChange={(value: any) => handleCustomFieldChange(field, value)}
```

Comment:
Is it possible to properly cast this instead of `(value: any)`?

Reply:
Type specification is not needed in this case, the function which accept value is specificed it as string.